starknet_py/net/schemas/rpc.py

```diff
@@ -218,7 +225,7 @@ class StarknetBlockSchema(Schema):
     status = BlockStatusField(data_key="status", required=True)
     root = NonPrefixedHex(data_key="new_root", required=True)
     transactions = fields.List(
-        fields.Nested(TypesOfTransactionsSchema(unknown=EXCLUDE)),
```

Comment:
I feel like preferably we would not have any `unknown=EXCLUDE` in the library?
It's usually a sign that something has been omitted I think

Reply:
You're right, I left it as it was, will remove that